rskj-core/src/test/java/co/rsk/peg/BridgeSupportTestUtil.java

```diff
@@ -22,6 +24,53 @@ public static Repository createRepository() {
         return new MutableRepository(new MutableTrieCache(new MutableTrieImpl(null, new Trie())));
     }
 
+    public static PartialMerkleTree createValidPmtWithTransactions(List<Sha256Hash> hashesToAdd, NetworkParameters networkParameters) {
+        byte[] bits = new byte[1];
+        bits[0] = 0x3f;
+
+        return new PartialMerkleTree(networkParameters, bits, hashesToAdd, 1);
+    }
+
+    public static void recreateChainAtHeightWithStoredBlockWithPmtAtHeight(
```

Comment:
nit: maybe a bit more redable?
```suggestion
    public static void recreateChainWithStoredBlockAndPmtAtHeight(
```

Reply:
i get the point, i just wanted to be clear about the fact that the block we want to store will be at one height, and the chain at other (equal or greater than the block height). But you are right, i will modify the name to make it more readable. Thanks :)